pkg/common/cns-lib/volume/listview.go

```diff
@@ -179,6 +184,31 @@ func (l *ListViewImpl) isClientValid() error {
 	return nil
 }
 
+// readConfig will ensure we read the latest credentials from the config secret
+// before attempting to create a new client
+func readConfig(ctx context.Context, vc *cnsvsphere.VirtualCenter) error {
+	log := logger.GetLogger(ctx)
+	cfg, err := config.GetConfig(ctx)
+	if err != nil {
+		log.Errorf("failed to read config. Error: %+v", err)
+		return err
+	}
+
+	newVCConfig, err := cnsvsphere.GetVirtualCenterConfig(ctx, cfg)
```

Comment:
Is list-view `readConfig` specific to single VC deployments ?
Why are we not using GetVirtualCenterConfigs here ?

Reply:
```
	vCenterIPs, err := GetVcenterIPs(cfg) //  make([]string, 0)
	if err != nil {
		return nil, err
	}
	host := vCenterIPs[0]
```
GetVirtualCenterConfig() is only getting the first config. I think I can remove readConfig() and call readVCConfigs() directly everywhere.